pkgs/tools/package-management/lix/common.nix

```diff
@@ -260,7 +260,7 @@ let
     passthru = {
       inherit aws-sdk-cpp boehmgc;
       tests = {
-        misc = nixosTests.misc.lix.passthru.override { nixPackage = self; };
+        misc = nixosTests.nix-misc.lix.passthru.override { nixPackage = self; };
```

Comment:
@RaitoBezarius This doesn't work before or after this PR or at 3388179c48fa673ae466f12d15cd7864b8e7cd3c when the line was added

```
$ nix build ".#lix.tests.misc"
error: attribute 'passthru' missing
```

The similarly patterned spark works

```
nix-repl> nixosTests.nix-misc.default
«derivation /nix/store/4qvam2nhs4dy5cibswrj1pici8s32h2p-vm-test-run-misc.drv»

nix-repl> nixosTests.spark.default
{ passthru = { ... }; recurseForDerivations = true; sparkCluster = «derivation /nix/store/ym7yki4znrln73ng96rj4fl7rsz95v99-vm-test-run-spark.drv»; }
```

https://github.com/NixOS/nixpkgs/blob/master/nixos/tests/spark/default.nix

Weird...

Reply:
This is because `discoverTests` will shortcut as soon as it sees the `test` attribute.